websub-generic-integration-tests/Ballerina.toml

```diff
@@ -1,40 +1,40 @@
 [package]
 org = "ballerina"
 name = "websub_generic_integration_tests"
-version = "@toml.version@"
+version = "1.0.9"
```

Comment:
Seems this changed file is mistakenly committed.

Reply:
Yes fixed it